expo/scheduler/views.py

```diff
@@ -630,18 +499,22 @@ def manage_volunteer_opportunities(request, event_id):
     event = get_object_or_404(Event, id=event_id)
     if 'create' in request.POST.keys() or 'duplicate' in request.POST.keys():
         if 'create' in request.POST.keys():
-            form = VolunteerOpportunityForm(request.POST, prefix='new_opp')
+            form = VolunteerOpportunityForm(
+                request.POST,
+                prefix='new_opp',
+                conference=get_current_conference())
         else:
-            form = VolunteerOpportunityForm(request.POST)
+            form = VolunteerOpportunityForm(
+                request.POST,
+                conference=get_current_conference())
         if form.is_valid():
             opp = form.save(commit=False)
             opp.type = "Volunteer"
             opp.save()
             data = form.cleaned_data
-            day = data.get('day')
-            time = data.get('time')
-            day = ' '.join([day.split(' ')[0], time])
-            start_time = datetime.strptime(day, "%Y-%m-%d %H:%M:%S")
+            day = data.get('day').day
+            time_parts = map(int, data.get('time').split(":"))
+            start_time = datetime.combine(day, dttime(*time_parts))
```

Comment:
I'm not sure if this is the right place... but when I do:
http://localhost:8282/scheduler/edit/Show/1328
- add a volunteer opportunity

Then the "day" of the opportunity does not show up correctly in the form when it loads.